graphql/host_resolver.go

```diff
@@ -58,6 +58,15 @@ func (r *hostResolver) HomeVolume(ctx context.Context, obj *restModel.APIHost) (
 	return nil, nil
 }
 
+// SleepSchedule is the resolver for the sleepSchedule field.
+func (r *hostResolver) SleepSchedule(ctx context.Context, obj *restModel.APIHost) (*host.SleepScheduleInfo, error) {
+	h, err := host.FindOne(ctx, host.ById(*obj.Id))
```

Comment:
I felt like that may be overkill since afaik there is no plan to include this data in any REST routes. I have actually wondered about this before though and would be interested in an app engineer's perspective!

Reply:
I don't think there's any reason we can't include it in the rest routes, if that would be preferable to having a separate resolver.

Reply:
In my opinion the separate resolver is preferable since it doesn't require duplicating the `SleepScheduleInfo` type to create `APISleepScheduleInfo` 🤷‍♀️